doc/docs/dev-contributing.md

```diff
@@ -15,7 +15,8 @@ proceed on the [Gitter channel](https://gitter.im/coursier/coursier).
       `git submodule update --init`  which will clone down the necessary
       submodules for the project.
   - Use the local `./sbt` rather than your machine sbt as it behaves slightly
-      different than the default sbt launcher.
+      different than the default sbt launcher. You get the following compile
+      error if you use the machine sbt: `not found: value sbtCoursierVersion`.
```

Comment:
I got the compile error below
```
/Users/lgeirsson/dev/coursier/build.sbt:485: error: not found: value sbtCoursierVersion
        "PLUGIN_VERSION" -> sbtCoursierVersion,
                            ^
```
and didn't find any relevant results when I grepped for "sbtCoursierVersion".